app/models/file_set.rb

```diff
@@ -4,4 +4,8 @@ class FileSet < Valhalla::Resource
   attribute :id, Valkyrie::Types::ID.optional
   attribute :title, Valkyrie::Types::Set
   attribute :member_ids, Valkyrie::Types::Array
+
+  def thumbnail_id
+    @thumbnail_id ||= Valkyrie.config.metadata_adapter.query_service.find_members(resource: self).find { |x| x.use.include?(Valkyrie::Vocab::PCDMUse.ServiceFile) }.try(:id)
```

Comment:
This took me a second to figure out before understanding that we are finding the pcdmuse:ServiceFile which the image server is using to generate the thumbnail.  Maybe we should move the image API suffix in here instead of returning the service file from `thumbnail_id`?